backend/src/packages/chats/chatbot.service.ts

```diff
@@ -0,0 +1,24 @@
+import { type OpenAi } from '#libs/packages/open-ai/open-ai.js';
+
+import { NO_RESPONSE_MESSAGE } from './libs/constants/constants.js';
+import { type OpenAiMessageGenerateRequestDto } from './libs/types/types.js';
+
+class ChatbotService {
+  private openAiService: OpenAi;
+  //TODO: add chat service
+  public constructor({ openAi }: { openAi: OpenAi }) {
+    this.openAiService = openAi;
+  }
+
+  public async generateReply({
+    content,
+  }: OpenAiMessageGenerateRequestDto): Promise<string | null> {
```

Comment:
```suggestion
  }: OpenAiMessageGenerateRequestDto): Promise<string> {
```
`reply ?? NO_RESPONSE_MESSAGE;` will always be a string

Reply:
done